README.md

```diff
@@ -189,6 +190,23 @@ as a pre-commit.
 
 **NOTE**: the `protect` command can only be used on git repos, running `protect` on files or directories will result in an error message.
 
+### Creating a baseline
+
+When scanning large repositories or repositories with a long history, it can be convenient to use a baseline. When using a baseline, 
+gitleaks will ignore any old findings which is present in the baseline. To create a baseline, run gitleaks with the `--report-path` parameter. 
+
+```
+gitleaks detect --report-path .gitleaksbaseline # This will save the report in a file called .gitleaksbaseline
+```
+
+***NOTE***: If you decide to save the baseline within the source three, you must add it to the .gitleaksignore file. Otherwise, the contents of this file will generate new issues.
```

Comment:
```suggestion
***NOTE***: If you decide to commit the baseline file to the repo, you must add it to the .gitleaksignore file. Otherwise, the contents of the baseline file will generate new issues.
```